web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -1891,13 +1957,18 @@ form#views-exposed-form-job-board-page {
 }
 
 .checkout-pane-order-summary {
-  background-color: #2A556B;
+  background-color: #FFFFFF;
   padding-bottom: 16px;
+  padding-left: 24px;
 
   .field-type-commerce-price {
     padding-left: 25px;
   }
 
+  td, td:first-child, td:last-child {
```

Comment:
Are the first- and last- child needed here? They might be to override other styles.